q0_linac.py

```diff
@@ -709,14 +722,16 @@ def takeNewCalibration(self, jt_search_start: datetime = None,
         
         self.heater_power = self.valveParams.refHeatLoadDes
         
-        print("Restoring initial cryo conditions")
```

Comment:
I don't think it hurts to have the info printed here? if you are still doing a 'restore' in the next line

Reply:
Info like, what it's doing?

Reply:
yea looks like you're removing the print statement? I think it can stay in?

Reply:
Oh because that gets printed in the lcld-tools function!